partials/template.blade.php

```diff
@@ -1,53 +1,127 @@
+<!-- This template requires bootstrap and style.css file to properly render. Make sure the parent file of this partial has bootstrap and assets/css/style.css included -->
+
 <div class="col-md-12 ts-top-banner">
-	<!--
-	<h1>Hero Image</h1>
-	<h2>This is a simple hero unit, a simple jumbotron-style component for calling extra 7
-	attention to featured content or information.</h2>
-	<p><a class="btn btn-primary btn-large">Learn more</a></p>
-	-->
+<!--
+<h1>Hero Image</h1>
+<h2>This is a simple hero unit, a simple jumbotron-style component for calling extra 7
+attention to featured content or information.</h2>
+<p><a class="btn btn-primary btn-large">Learn more</a></p>
+-->
 </div>
 
-
 <div class="col-md-12 breathing-space">
 
 	<div class="row">
-		<div class="col-md-offset-2 col-md-8">
-			<h1 >Why Supercomputing?</h1>
-			<p style="color:#cccccc;">
-				Supercomputing simplifies and boosts modeling simulation, helps to make better use of computational expertise and create manufacturing apps.<br/>
-			</p>
-			<hr/>
+		<div class="col-md-6">
+			<p><strong>Blue Cluster</strong> (Originally known as Orion)</p>
+			<!-- Please use img-fluid instead of img-responsive if your bootstrap version if >=4.0 -->
+			<img class="center-image img-responsive" src="/assets/img/orion.png">
```

Comment:
I think they want to link to the live image, not a static one. Same deal on line 83.